Thesis.tex

```diff
@@ -286,12 +286,14 @@ \subsection{Against IP}
 To the author of this chapter, the cost of CSS to the advancement of the sciences is too big and the benefit too small to accept copyrights in the area of software creation. As mentionned in \ref{ssec:OSS}, it is possible to make money (make lots of money) when working with open source software.
 
 \subsection{Addressing Against IP's points}
-One of the key advantages of open-source software is the ability to modify the code. But many users don't have the need, let alone the programming skills to so. Even if modifying the codes is desired doing so always incurs costs, at the very least in work hours. A developer needs to learn the project structure, learn naming and coding conventions, understand where the extensibility points are and often have to learn about additional open-source libraries. All this before actually writing the code for the desired modification. 
+One of the key advantages of open-source software is the ability to modify the code. But many users don't have the need, let alone the programming skills to so. Even if modifying the code is desired doing so always incurs costs, at the very least in work hours. A developer needs to learn the project structure, learn naming and coding conventions, understand where the extensibility points are and often have to learn about additional open-source libraries. All this before actually writing the code for the desired modification. 
 
 Furthermore these changes then need to be maintained. With every new release of the underlying software changes made need to be tested and often modified. Otherwise the user runs the risk of relying on outdated, potentially buggy or insecure software just because maintaining customizations is not profitable enough. This adds risks to the software used and potentially projects depending on it.
  
 A major part of the incentive for a developer to work on a open-source project is to gain recognition as the author. For very small, specialized markets this incentive diminishes drastically. 
 
+Of course there are a lot of open source projects with big communities of developers and users, as the afore mentioned free operating systems. But there are areas where the market is so small that it is unlikely that such a community will form, like the broadcast industry. Its users are not developers so the cannot help write the software they need. Also for those users need easy to learn and easy to use software that helps them achieve the productivity they need. Training and ease of use are often lacking with open source projects, especially small ones.
```

Comment:
_Furthermore, _ those users need _software that is easy to learn and use_, and helps them achieve the productivity they need.